<launch>
  <arg name="node_name" default="sick_S3x_driver" />
  <node pkg="sick_s3x" type="sick_s3x_node" name="$(arg node_name)" output="screen">
    <param name="model" value="S300" />
    <param name="reduced_fov" value="0" />
    <param name="devicename" value="/dev/ttyUSB0" />
    <param name="baudrate" value="500000" />
    <param name="tf_x" value="0.71" />
    <param name="tf_y" value="0.015" />
    <param name="tf_z" value="0.54" />
    <param name="tf_roll" value="0.0" />
    <param name="tf_pitch" value="0.0" />
    <param name="tf_yaw" value="0.0146" />
    <param name="base_link" value="/state_base_link" />
    <param name="laser_flipped" value="1" />
  </node>
</launch>
